core/templates/pages/blog-dashboard-page/blog-dashboard-page.mainpage.html

```diff
@@ -13,20 +13,21 @@
   </head>
   <body>
     <oppia-root>
-      <div ng-controller="Base">
+      <div ng-controller="Base" dir="<[direction]>">
         <oppia-base-content>
           <navbar-breadcrumb>
-            <oppia-blog-dashboard-navbar-breadcrumb>
-            </oppia-blog-dashboard-navbar-breadcrumb>
+            <oppia-blog-dashboard-navbar>
+            </oppia-blog-dashboard-navbar>
```

Comment:
I suggest the breadcrumb suffix is better.

Reply:
Sure.